.github/workflows/verify-pr-commit.yml

```diff
@@ -271,7 +271,7 @@ jobs:
           profile: minimal
           toolchain: stable
       - name: Check
-        run: SKIP_WASM_BUILD= cargo check --all-features
+        run: SKIP_WASM_BUILD= cargo check --features runtime-benchmarks,all-frequency-features,std
```

Comment:
`--all-features` doesn't work now due to the mutually exclusive nature of the features.